model-optimizer/mo/utils/ir_reader/extenders/strided_slice_extender.py

```diff
@@ -12,8 +14,12 @@ class StridedSlice_extender(Extender):
 
     @staticmethod
     def extend(op: Node):
+        input_shape = op.in_port(0).data.get_shape()
         for attr in StridedSlice.get_mask_names():
```

Comment:
You need to check that value is set for `begin_mask` and `end_mask` attributes. Otherwise, the MO IR Reader must report that IR is incorrect.

Reply:
Done.